### PR TITLE
DWR-1220 Changed aggregate report query to use ids instead of codes.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: aggregate-service/src/test/java/org/opentestsystem/rdw/olap/repository/impl/JdbcCustomAggregateReportRowRepositoryRST.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public void setUp() {
         districtsCount = countRowsInTable(template, "district");
 
         queryBuilder = ReportQuery.builder()
-                .asmtGrades(of(-8))
+                .asmtGradeIds(of(-8))
```
Comment (esotrope) [inline aggregate-service/src/test/java/org/opentestsystem/rdw/olap/repository/impl/JdbcCustomAggregateReportRowRepositoryRST.java]: the abbr kills my ocd ; p

Comment (agorina) [reply aggregate-service/src/test/java/org/opentestsystem/rdw/olap/repository/impl/JdbcCustomAggregateReportRowRepositoryRST.java]: I know, you did mention it before :) 
I really really do not like spelling out the loooooong assessment ...sorry...

Comment (esotrope) [reply aggregate-service/src/test/java/org/opentestsystem/rdw/olap/repository/impl/JdbcCustomAggregateReportRowRepositoryRST.java]: but you have `assessmentType` in the same object even what gives? x D

Comment (chdietz) [reply aggregate-service/src/test/java/org/opentestsystem/rdw/olap/repository/impl/JdbcCustomAggregateReportRowRepositoryRST.java]: we seem to have standardized on `asmt` for assessment.

**Pull request merged**